test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -13,11 +13,18 @@ def setup
     @three_ds_credit_card = credit_card('4000000000003220',
       verification_value: '737',
       month: 10,
-      year: 2020)
+      year: 2021
+    )
```

Comment:
nitpick - i'm getting a couple of rubocop errors around these closing parens (`Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.`)